lms/templates/discussion/index.html

```diff
@@ -32,14 +33,14 @@
          data-course-id="${course_id}"
          data-course-name="${course.display_name_with_default}"
          data-user-info="${user_info}"
-         data-user-create-comment="${can_create_comment}"
-         data-user-create-subcomment="${can_create_subcomment}"
+         data-user-create-comment="${can_create_comment | n, dump_js_escaped_json}"
+         data-user-create-subcomment="${can_create_subcomment | n, dump_js_escaped_json}"
```

Comment:
Oh. In general, data attributes are just regular HTML, and should not have the added filters. These lines should probably be reverted. I explain more in the read the docs for safe templates.  Thanks.